src/GeositeFramework/plugins/layer_selector_v2/layers.json

```diff
@@ -150,5 +150,43 @@
                 "downloadUrl": "http://www.tceq.texas.gov/assets/public/gis/exports/au_oysterwaters.zip"
             }
         ]
+    },
+    {
+        "displayName": "WMS Example Layers",
+        "server": {
+            "type": "wms",
+            "layerType": "dynamic",
+            "url": "http://sampleserver1.arcgisonline.com/ArcGIS/services/Specialty/",
+            "name": "ESRI_StatesCitiesRivers_USA"
+        },
+        "includeLayers": [
+            {
+                "name": "States"
+            },
+            {
+                "name": "Cities"
+            },
+            {
+                "name": "Rivers"
+            }
+        ]
+    },
+    {
+        "displayName": "Global Risk Data Platform",
```

Comment:
The legends for this map service won't load. Should we include this in our sample layers?

![image](https://cloud.githubusercontent.com/assets/43062/13267738/e8785a2c-da4c-11e5-9519-31681c0f258f.png)